### PR TITLE
[Tizen] Remove the unused code from Application storage

### PR DESCRIPTION
**Pull request by pozdnyakov**
SSIA

### CODE REVIEW

**Comment by pozdnyakov** [comment]
@kenchris PTAL

File: application/common/tizen/application_storage.cc [hunk 1]
```diff
@@ -4,51 +4,149 @@
...
 }
 
 bool ApplicationStorage::GetInstalledApplicationIDs(
-    std::vector<std::string>& app_ids) const {  // NOLINT
-  return impl_->GetInstalledApplicationIDs(app_ids);
+  std::vector<std::string>& app_ids) const {  // NOLINT
+  uid_t uid = getuid();
+
+  for (size_t i = 0; i < arraysize(kXWalkPackageTypes); ++i) {
+    int count = 0;
```
Comment (shaochangbin) [inline application/common/tizen/application_storage.cc]: drive by. 
Put 'count' declaration to the place it's used.

File: application/common/tizen/application_storage.cc [hunk 2]
```diff
@@ -4,51 +4,149 @@
...
+    if (ret != AIL_ERROR_OK) {
+      LOG(ERROR) << "Failed to init AIL filter.";
+      ail_filter_destroy(filter);
+      return false;
+    }
+
+    if (uid != GLOBAL_USER)
+      ret = ail_filter_count_usr_appinfo(filter, &count, uid);
+    else
+      ret = ail_filter_count_appinfo(filter, &count);
```
Comment (shaochangbin) [inline application/common/tizen/application_storage.cc]: ret = (uid != GLOBAL_USER) ? ail_filter_count_usr_appinfo(filter, &count, uid)
                                                : ail_filter_count_appinfo(filter, &count);

**Comment by takethathe** [comment]
LGTM

**Comment by rakuco** [comment]
Rendered obsolete by #3299

**Pull request closed without merging**